tests/policy_as_code_test.go

```diff
@@ -0,0 +1,242 @@
+//go:build test_e2e
+
+package tests
+
+import (
+	"bytes"
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/config-controller/api/v1alpha1"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/testutils/centralgrpc"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"gopkg.in/yaml.v3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/cache"
+)
+
+func TestPolicyAsCode(t *testing.T) {
+	suite.Run(t, new(PolicyAsCodeSuite))
+}
+
+type PolicyAsCodeSuite struct {
+	suite.Suite
+	centralClient   v1.PolicyServiceClient
+	k8sClient       dynamic.ResourceInterface
+	informerfactory dynamicinformer.DynamicSharedInformerFactory
+	informer        informers.GenericInformer
+	policies        []*storage.Policy
+	ctx             context.Context
+	cleanupCtx      context.Context
+	cancel          func()
+	stopCh          chan struct{}
+}
+
+func (pc *PolicyAsCodeSuite) SetupSuite() {
+	pc.ctx, pc.cleanupCtx, pc.cancel = testContexts(pc.T(), "TestPolicyAsCode", 15*time.Minute)
+
+	conn := centralgrpc.GRPCConnectionToCentral(pc.T())
+	pc.centralClient = v1.NewPolicyServiceClient(conn)
+
+	dynamicClient := dynamic.NewForConfigOrDie(getConfig(pc.T()))
+	pc.k8sClient = dynamicClient.Resource(schema.GroupVersionResource{
+		Group:    "config.stackrox.io",
+		Version:  "v1alpha1",
+		Resource: "securitypolicies",
+	}).Namespace("stackrox")
```

Comment:
use namespace env var instead?

Reply:
It doesn't look like any other e2e tests use the ROX_NAMESPACE env var, and so far in my testing `stackrox` has worked without issue.